### PR TITLE
Unify hydra and non-hydra runs, add cfg entrypoint, add plotly test

### LINKED ISSUE #133: Ensure plotly plots work correctly

### PR DESCRIPTION
**Pull request by timkpaine**
fixes #133

### CODE REVIEW

File: examples/inline.yaml [hunk 1]
```diff
@@ -131,7 +131,7 @@ content:
 
   - _target_: nbprint.ContentCode
     content: |
-      from perspective import PerspectiveWidget
+      from perspective.widget import PerspectiveWidget
```
Comment (ptomecek) [inline examples/inline.yaml:134]: I did not realize this was supported. Will it properly show all the different perspective chart types?

File: nbprint/cli.py [hunk 2]
```diff
@@ -3,33 +3,58 @@
...
+    # prune any empty strings
+    overrides = [o for o in overrides if o]
+
     with initialize_config_dir(config_dir=str(path.parent.absolute()), version_base=None):
-        cfg = compose(config_name=str(path.name), overrides=overrides)
-        config = instantiate(cfg)
+        hydra_config = compose(config_name=str(path.name), overrides=overrides)
+
+        # bridge hydra and non-hydra
+        extras = {"name": path.name.replace(".yaml", "").replace(".yml", "")} if "name" not in hydra_config else {}
```
Comment (ptomecek) [inline nbprint/cli.py:42]: Nice

**Review by ptomecek — APPROVED** [review]

File: nbprint/models/plotly.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+from typing import Optional
+
+from IPython.display import HTML
+
+from nbprint import Content
+
+__all__ = ("PlotlyRendererConfiguration",)
+
+
+class PlotlyRendererConfiguration(Content):
```
[the remark this answers is not in the document]
Comment (timkpaine) [reply nbprint/models/plotly.py:11]: resolved by cheating and using an env var

File: examples/inline.yaml (hunk 1 above)
Comment (timkpaine) [reply examples/inline.yaml:134]: this is for `perspective>=3` support, but should work with either

**Pull request merged**